pages/spotlight/[slug]/compare.tsx

```diff
@@ -18,16 +20,32 @@ interface CompareProps {
 }
 
 const Compare: NextPage<CompareProps> = ({ setData, scaffold, page }) => {
+  const router = useRouter();
   useEffect(() => {
     if (setData) {
       setData({ ...scaffold, title: page.title, slug: getSlugFromURL(page.relative_url) });
     }
   }, [setData, scaffold]);
+
+  const getQueryLocation = (): SpotlightLocation | undefined => {
+    if (router.query.ln && router.query.lc) {
+      return {
+        name: router.query.ln.toString(),
+        geocode: router.query.lc.toString()
```

Comment:
ln & lc could be arrays ...

Reply:
Updated this to receive a comma separated string with multiple locations